tests/integration_tests/tests/agentless_tests/test_snapshot.py

```diff
@@ -383,6 +387,55 @@ def test_v_4_6_0_restore_snapshot_and_restart_services(self):
         docl.execute('cfy_manager restart --force')
         self.assertTrue(self._all_services_restarted_properly())
 
+    def test_snapshot_status_returns_correct_status(self):
+        self._assert_restore_marker_file_does_not_exist()
+        self._assert_snapshot_restore_status(is_running=False)
+
+        snapshot_id = "test_" + str(uuid4())
+        snapshot_create_execution = self.client.snapshots.create(
+            snapshot_id, False)
+        self.wait_for_execution_to_end(snapshot_create_execution)
+
+        snapshot_restore_execution_id = self.client.snapshots.restore(
```

Comment:
this can not create a flaky test, because the restore could finish fast?
maybe some we need to control it more